apps/studio/src/pages/home/SideBar.tsx

```diff
@@ -177,7 +178,9 @@ export const SideBar: FunctionComponent<SideBarProps> = (
         px={2.5}
         width="100%"
       >
-        <NEWMLogoSmInverse />
+        <Link href={NEWM_IO_URL} target="_blank" rel="noopener">
+          <NEWMLogo height="40" width="40" />
```

Comment:
There is an issue I can't identify the fix for, where the `linearGradient`, NEWM multi colored circle around the icon, isn't visible in the mobile view sidebar. I have tracked it down to the `Drawer` component and the `variant="temporary` setting but haven't found anything leading to a solution online. I will add a notice on the task and create a separate low priority bug for it.

Reply:
Looked into this deeper and looks like it is an issue with the SVG ID being used within the `linearGradient` and how  the `Drawer` component must render SVGs causing a conflict. 

I will look into creating SVGs with a dynamic hash for the ID but if it's too much effort for this task I will sideline it to a bug.
https://stackoverflow.com/questions/14773940/conflicting-ids-in-two-different-svg-files-included-in-the-same-html-document